src/zappy_server_src/core/entity/pack.c

```diff
@@ -0,0 +1,114 @@
+/*
+** EPITECH PROJECT, 2022
+** ZAPPYMIRROR
+** File description:
+** unpack
+*/
+
+/// \file src/zappy_server_src/core/entity/pack.c
+
+#include "entity.h"
+#include "entity_types.h"
+#include "container.h"
+#include "player.h"
+#include "tile.h"
+#include "eggs.h"
+#include <string.h>
+#include <sys/queue.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+char *pack_container(container_t *cont)
+{
+    char *tmp = (char*)malloc(sizeof(char) * (get_len_container(cont) + 21));
```

Comment:
Why 21 ? Use define

Reply:
Thinking 21 is the total length need to be allocate to use strcat afterall

Better using a function in case of this length increases or decreases in the futur